components/ws-manager-bridge/src/bridge-controller.ts

```diff
@@ -19,20 +19,13 @@ import { Metrics } from "./metrics";
 
 @injectable()
 export class BridgeController {
-    @inject(Configuration)
-    protected readonly config: Configuration;
-
-    @inject(WorkspaceManagerBridgeFactory)
-    protected readonly bridgeFactory: interfaces.Factory<WorkspaceManagerBridge>;
-
-    @inject(WorkspaceManagerClientProvider)
-    protected readonly clientProvider: WorkspaceManagerClientProvider;
-
-    @inject(WorkspaceClusterDB)
-    protected readonly db: WorkspaceClusterDB;
-
-    @inject(Metrics)
-    protected readonly metrics: Metrics;
+    constructor(
+        @inject(Configuration) private readonly config: Configuration,
+        @inject(WorkspaceManagerBridgeFactory)
+        private readonly bridgeFactory: interfaces.Factory<WorkspaceManagerBridge>,
+        @inject(WorkspaceManagerClientProvider) private readonly clientProvider: WorkspaceManagerClientProvider,
+        @inject(Metrics) private readonly metrics: Metrics,
+    ) {}
 
     protected readonly bridges: Map<string, WorkspaceManagerBridge> = new Map();
```

Comment:
We can also generally replace all usages of `protected` with `private` as we are not using inheritance usually. This often also reveals that methods or fields are actually unused.